airflow/migrations/versions/0123_2_5_0_add_display_name_for_dag_and_task_.py

```diff
@@ -0,0 +1,48 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add display name for dag and task instance
+
+Revision ID: d83579315023
+Revises: 290244fb8b83
+Create Date: 2022-12-06 20:26:07.521273
```

Comment:
Was this file generated with Alembic?
https://github.com/apache/airflow/blob/0c798252efb83844a2e0ba08d74b32eed2e317ad/CONTRIBUTING.rst#metadata-database-updates

The date + file name suggests it was manually copied from another migration file

Reply:
To be honest I'm not sure.
It's from the previous PR https://github.com/apache/airflow/pull/28183 and I'm trying to conintue the unfinished work.

Is there anything wrong in this file? I can try to fix.

Reply:
I updated the file